verified-answers.yaml

```diff
@@ -14,3 +16,1551 @@
       title: MongoDB Atlas Registration
     - url: https://www.mongodb.com/basics/mongodb-atlas-tutorial?tck=mongodb_ai_chatbot
       title: MongoDB Atlas Tutorial
+- answer: >-
+    To get started with MongoDB, you can try the 5-minute interactive tutorial
+    that connects you to a MongoDB Atlas cluster and walks you through
+    inserting, querying, and deleting data in MongoDB. You do not need to
+    install anything for this tutorial. Simply click the Launch button of the
+    in-browser Integrated Development Environment to start the tutorial.
+
+    After completing the tutorial, you can also deploy a free cluster without any installation overhead by following the instructions in the "Getting Started with Atlas" documentation.
+
+    Let me know if you have any other questions!
+  author_email: ian.fogelman@mongodb.com
+  questions:
+    - Get started with MongoDB
+  references:
+    - url: https://mongodb.com/docs/manual/tutorial/getting-started/
```

Comment:
Just confirming - we don't have anything planned atm to check that verified answer references are up-to-date, don't 404, don't redirect, etc, right?

Might be something to capture in jira / a good intern project

Reply:
https://jira.mongodb.org/browse/EAI-342